hack/install-deps.sh

```diff
@@ -42,7 +42,7 @@ go get -d ${RUNC_PKG}/...
 cd ${GOPATH}/src/${RUNC_PKG}
 git fetch --all
 git checkout ${RUNC_VERSION}
-make
+make BUILDTAGS='seccomp apparmor'
```

Comment:
yeah was going to add the selinux option as well. 

We should probably discuss this.